wavefront/server/apps/call_processing/call_processing/services/tool_wrapper_service.py

```
@@ -0,0 +1,381 @@
+                    result_callback = None
+
+                # Execute the tool wrapper
+                result = await wrapper.execute(tool_params)
+
+                logger.info(f"Tool '{tool_name}' execution result: {result}")
+
+                # Return result via callback if available
+                if result_callback:
+                    await result_callback(result)
+
+                return result
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)